### PR TITLE
Add ActivityLogger plugin

### PR DESCRIPTION
**Pull request by ishanvyas22**
Add https://github.com/elstc/cakephp-activity-logger

### CODE REVIEW

**Comment by dereuromark** [comment]
Your plugin name looks like a lie - as per our guidelines
`Elastic/ActivityLogger` seems to be the name.

File: README.md [hunk 1]
```diff
@@ -96,6 +96,7 @@ Additional lists you might find useful:
 ## Auditing / Logging
 *Plugins for auditing and logging.*
 
+- [Activity Logger plugin](https://github.com/elstc/cakephp-activity-logger) - ActivityLogger plugin for CakePHP.
```
Comment (dereuromark) [inline README.md:99]: The description could explain a bit more, and the "plugin for CakePHP" can be dropped as this is obvious.

**Comment by dereuromark** [comment]
Code comments in non English is not something that fits our quality criteria for reusability in the community ( https://github.com/elstc/cakephp-activity-logger/blob/master/src/Model/Behavior/LoggerBehavior.php#L217 ).

**Comment by dereuromark** [comment]
In what way is that different/better than the existing AuditStash plugin?
https://github.com/FriendsOfCake/awesome-cakephp#auditing--logging
If this was for 4.x this might be more interesting, but it also seems to be only for 3.x still.

**Comment by ishanvyas22** [comment]
> In what way is that different/better than the existing AuditStash plugin?
> https://github.com/FriendsOfCake/awesome-cakephp#auditing--logging
> If this was for 4.x this might be more interesting, but it also seems to be only for 3.x still.

Yes @dereuromark , you are right. AuditStash plugin is better than ActivityLog, when I looked the AuditStash plugin I was in impression that it will only store data into elastic search(that I don't want). But it also stores activity data into mysql database table, and also well maintained and supports cake 4.x.

So closing the merge request as we don't need new "not-maintained" plugin in awesome list.

**Pull request closed without merging**